api-testing/multicloud-api/README.md

```diff
@@ -0,0 +1,27 @@
+# API Testing Automation Framework
+
+**Introduction**
+
+This framework has developed in [JUNIT](https://junit.org/) using [Gradle](https://gradle.org/) Build tool.
+Currently added for multicloud project of soda foundation. Later it could be reused for other projects.
+Junit framework details [Link](https://www.tutorialspoint.com/junit/index.htm)
+
+**Why this?**
+
+Junit is widely used testing framework along with Java Programming Language. You can use this automation framework for both unit testing and UI testing.
+
+**How to install and use?**
```

Comment:
Is there any requirements for this framework, for example, JDK must be higher than 1.8. Please add an explanation here if need.

Reply:
I have already added Junit Link in Introduction.

Reply:
@Click2cloud-Wadhai  @sfzeng is asking about the JAVA version. Not the JUnit.
Do you agree this is a JAVA project, then which versions of JDK is supported. Or JDK is a prerequisites or not for this project .If yes, then which version of JDK should the user/developer install for this project?

Reply:
I have mentioned. Please review.